packages/graphql-auth-transformer/src/__tests__/OperationsArgument.test.ts

```diff
@@ -0,0 +1,106 @@
+import GraphQLTransform from 'graphql-transformer-core'
+import { ResourceConstants } from 'graphql-transformer-common'
+import { DynamoDBModelTransformer } from 'graphql-dynamodb-transformer'
+import { ModelAuthTransformer } from '../ModelAuthTransformer'
+
+test('Test "read" auth operation', () => {
+    const validSchema = `
+    type Post @model @auth(rules: [{allow: groups, groups: ["Admin", "Dev"], operations: [read]}]) {
+        id: ID!
+        title: String!
+        createdAt: String
+        updatedAt: String
+    }
+    `
+    const transformer = new GraphQLTransform({
+        transformers: [
+            new DynamoDBModelTransformer(),
+            new ModelAuthTransformer({authMode: 'AMAZON_COGNITO_USER_POOLS'})
+        ]
+    })
+    const out = transformer.transform(validSchema)
+    expect(out).toBeDefined()
+    expect(
+        out.rootStack.Resources[ResourceConstants.RESOURCES.GraphQLAPILogicalID].Properties.AuthenticationType
+    ).toEqual('AMAZON_COGNITO_USER_POOLS')
+    expect(
+        out.resolvers['Query.getPost.res.vtl']
+    ).toContain('Authorization rule:')
```

Comment:
Do you think these would be good candidate for snapshot tests?

Reply:
Perhaps. Do you mean storing the output and then diffing them between runs?

Reply:
Yes. Jest has built in support for that. 
```typescript
expect(out.resolvers['Query.getPost.res.vtl']).toMatchSnapshot();
```
this should do it